packages/react/src/components/MenuButton/index.tsx

```diff
@@ -80,6 +80,17 @@ export interface MenuButtonProps extends ComponentProps<'div'> {
    * Specify the tabIndex of the button.
    */
   tabIndex?: number;
+
+  /**
+   * Specify a DOM node where the Menu should be rendered in. Defaults to document.body.
+   */
+  menuTarget?: Element;
+
+  /**
+   * Specify a DOM node where the Menu should be rendered in. Defaults to document.body.
+   * (Deprecated props)
+   */
+  target?: Element;
```

Comment:
Here I think it makes sense to only have one of these. Since it's referring to the target of the Menu, the name `menuTarget` feels appropriate.

```suggestion

  /**
   * Specify a DOM node where the Menu should be rendered in. Defaults to document.body.
   */
  menuTarget?: Element;
```

Reply:
I have made the suggested change.